src/main/docs/guide/breaks.adoc

```diff
@@ -2,23 +2,4 @@ This section documents breaking changes between Micronaut Neo4J versions:
 
 === Micronaut Neo4J {version}
 
-Micronaut Neo4J is based on Neo4j 5.x, which has breaking changes from Neo4j 3.5.x used by previous versions of Micronaut Neo4J.
-
-For more information, refer to the following Neo4j guides:
-
-- https://neo4j.com/docs/upgrade-migration-guide/current/[Neo4j Upgrade and Migration Guide]
-
-- https://neo4j.com/docs/upgrade-migration-guide/current/migration/surface-changes/[Breaking changes between Neo4j 3.5 and Neo4j 4.x]
-- https://neo4j.com/docs/upgrade-migration-guide/current/version-5/migration/breaking-changes/[Breaking changes between Neo4j 4.4 and Neo4j 5.x]
-
-Neo4J Java Driver 5.x adds compatibility for Java 17
-
-- `GraphDatabase.routingDriver()` method – used by api:neo4j.bolt.Neo4jBoltConfiguration[] – was deprecated in Neo4J 4.4.x and removed in 5.x. As a consequence, api:neo4j.bolt.Neo4jBoltConfiguration[] now only accepts a single URI, whereas it previously accepted a list of them as an option. For more information, see the Neo4J guide on
-https://neo4j.com/docs/javascript-manual/current/client-applications/#js-initial-address-resolution[Initial Address Resolution]
-
-Neo4J Harness 5.x adds compatibility for Java 17
-
-Starting with Micronaut Framework 4.0.0 the Neo4j Harness requires a runtime dependency on `javax.annotation` and `javax.validation`
-
-dependency:javax.annotation-api[version="1.3.2", groupId="javax.annotation", scope="testRuntimeOnly"]
-dependency:validation-api[version="2.0.1.Final", groupId="javax.validation", scope="testRuntimeOnly"]
+The `EmbeddedNeo4jServer` bean that supports using the Neo4j Harness was deprecated in Micronaut Neo4J 6.1.0 in favor of https://java.testcontainers.org/modules/databases/neo4j/[Testcontainers Neo4j]. Micronaut Neo4J 7.0.0 removed `EmbeddedNeo4jServer` and the relevant configuration properties in api:neo4j.bolt.Neo4jBoltConfiguration[]. In addition the `org.neo4j.test:neo4j-harness` librarty is no longer published with the Micronaut Neo4J BOM
```

Comment:
```suggestion
The `EmbeddedNeo4jServer` bean that supports using the Neo4j Harness was deprecated in Micronaut Neo4J 6.1.0 in favor of https://java.testcontainers.org/modules/databases/neo4j/[Testcontainers Neo4j].
Micronaut Neo4J 7.0.0 removed `EmbeddedNeo4jServer` and the relevant configuration properties in api:neo4j.bolt.Neo4jBoltConfiguration[].
In addition, the `org.neo4j.test:neo4j-harness` library is no longer published with the Micronaut Neo4J BOM
```

Reply:
Putting sentences on the next line (with no gap) means they are rendered as one contiguous block of text, but are easier to see in the editor (IMHO) 😉